next/components/forms/groups/InputSelectGroup.tsx

```diff
@@ -0,0 +1,163 @@
+
+  addNew,
+}: SelectFieldProps & InputBase & { addNew: string }) => {
+  const [fieldGroups, setFieldGroups] = useState([
+    { inputField: '', selectField: [], id: uuidv4() },
+  ])
+  const containerStyle = cx('flex flex-col items-start gap-4', {})
+
+  const addField = () => {
+    setFieldGroups([...fieldGroups, { inputField: '', selectField: [], id: uuidv4() }])
+  }
+
+  const removeField = (id: string) => {
+    setFieldGroups((current) => current.filter((element) => element.id !== id))
+  }
+
+  const updateState = (index: number) => (e: any) => {
+    const newArray = fieldGroups.map((item, i) => {
+      return index === i ? { ...item, inputField: e } : item
+    })
+    setFieldGroups(newArray)
+  }
+
+  return (
+    <div className={containerStyle}>
+      {fieldGroups.map((element, index) => {
+        return (
+          <div key={index} className="flex flex-row items-start gap-4 h-full">
+            <div className="flex relative flex-col items-start pt-3 w-3 ">
+              <div className="bg-gray-200 w-0.5 h-[50px]" />
+              <div className="w-3 h-0.5 bg-gray-200" />
+            </div>
+            <div className={InputClassName}>
+              <InputField
+                label={InputLabel}
+                placeholder={InputPlaceholder ?? ''}
+                description={InputDescription}
+                tooltip={InputTooltip}
+                leftIcon={InputLeftIcon}
+                required={InputRequired}
+                resetIcon={InputResetIcon}
+                errorMessage={InputErrorMessage}
+                disabled={InputDisabled}
+                value={element.inputField}
+                onChange={updateState(index)}
+              />
+            </div>
+            <SelectField
+              label={SelectLabel}
+              type={SelectType}
+              enumOptions={SelectEnumOptions}
+              disabled={SelectDisabled}
+              tooltip={SelectTooltip}
+              placeholder={SelectPlaceholder}
+              errorMessage={SelectErrorMessage}
+              description={SelectDescription}
+              required={SelectRequired}
+              explicitOptional={SelectExplicitOptional}
+              dropdownDivider={SelectDropdownDivider}
+              selectAllOption={SelectSelectAllOption}
+              value={element.selectField}
+              onChange={SelectOnChange}
+              className={SelectClassName}
+            />
+            {fieldGroups.length > 1 ? (
+              <div
+                className="flex w-6 h-6 justify-center items-center cursor-pointer mt-auto mb-5 rotate-45"
+                onClick={() => removeField(element.id)}
+              >
+                <PlusCircleIcon fill="red" />
+              </div>
+            ) : (
+              // 0 IQ solution, I will fix it during other fixes
+              <div className="w-6 h-6" />
+            )}
+          </div>
+        )
+      })}
+      <div className="py-0 pr-0 pl-8">
```

Comment:
once again py-0 pr-0 does not seem to do anything - please recheck the whole code for any like these